data/registers/canfd.yaml

```diff
@@ -0,0 +1,3284 @@
+  - name: RXF1A
+    description: MCAN Rx FIFO 1 Acknowledge.
+    byte_offset: 184
+    fieldset: RXF1A
+  - name: RXESC
+    description: MCAN Rx Buffer / FIFO Element Size Configuration.
+    byte_offset: 188
+    fieldset: RXESC
+  - name: TXBC
+    description: MCAN Tx Buffer Configuration.
+    byte_offset: 192
+    fieldset: TXBC
+  - name: TXFQS
+    description: MCAN Tx FIFO / Queue Status.
+    byte_offset: 196
+    access: Read
+    fieldset: TXFQS
+  - name: TXESC
+    description: MCAN Tx Buffer Element Size Configuration.
+    byte_offset: 200
+    fieldset: TXESC
+  - name: TXBRP
+    description: MCAN Tx Buffer Request Pending.
+    byte_offset: 204
+    access: Read
+    fieldset: TXBRP
+  - name: TXBAR
+    description: MCAN Tx Buffer Add Request.
+    byte_offset: 208
+    fieldset: TXBAR
+  - name: TXBCR
+    description: MCAN Tx Buffer Cancellation Request.
+    byte_offset: 212
+    fieldset: TXBCR
+  - name: TXBTO
+    description: MCAN Tx Buffer Transmission Occurred.
+    byte_offset: 216
+    access: Read
+    fieldset: TXBTO
+  - name: TXBCF
+    description: MCAN Tx Buffer Cancellation Finished.
+    byte_offset: 220
+    access: Read
+    fieldset: TXBCF
+  - name: TXBTIE
+    description: MCAN Tx Buffer Transmission Interrupt Enable.
+    byte_offset: 224
+    fieldset: TXBTIE
+  - name: TXBCIE
+    description: MCAN Tx Buffer Cancellation Finished Interrupt Enable.
+    byte_offset: 228
+    fieldset: TXBCIE
+  - name: TXEFC
+    description: MCAN Tx Event FIFO Configuration.
+    byte_offset: 240
+    fieldset: TXEFC
+  - name: TXEFS
+    description: MCAN Tx Event FIFO Status.
+    byte_offset: 244
+    access: Read
+    fieldset: TXEFS
+  - name: TXEFA
+    description: MCAN Tx Event FIFO Acknowledge.
+    byte_offset: 248
+    fieldset: TXEFA
+block/CANFD0_PROCESSORS:
+  items:
+  - name: SUBSYS_REGS
+    array:
+      len: 1
+      stride: 44
+    byte_offset: 0
+    block: SUBSYS
+  - name: ECC_REGS
+    array:
+      len: 1
+      stride: 528
+    byte_offset: 512
+    block: ECC
+block/CANFD0_TI_WRAPPER:
+  items:
+  - name: PROCESSORS
+    array:
+      len: 1
+      stride: 1040
+    byte_offset: 0
+    block: CANFD0_PROCESSORS
+  - name: MSP
+    array:
+      len: 1
+      stride: 512
+    byte_offset: 1536
+    block: MSP
+block/CPU_INT:
+  items:
+  - name: IIDX
+    description: Interrupt Index Register.
+    byte_offset: 0
+    access: Read
+    fieldset: IIDX
+  - name: IMASK
+    description: Interrupt mask.
+    byte_offset: 8
+    fieldset: IMASK
+  - name: RIS
+    description: Raw interrupt status.
+    byte_offset: 16
+    access: Read
+    fieldset: RIS
+  - name: MIS
+    description: Masked interrupt status.
+    byte_offset: 24
+    access: Read
+    fieldset: MIS
+  - name: ISET
+    description: Interrupt set.
+    byte_offset: 32
+    access: Write
+    fieldset: ISET
+  - name: ICLR
+    description: Interrupt clear.
+    byte_offset: 40
+    access: Write
+    fieldset: ICLR
```

Comment:
These fieldsets can be merged (see the `CPU_INT` fieldset merging for other peripherals)